### DISCUSSION TITLE
Provide deb packages

### DISCUSSION THREAD

**Discussion by hmsjy2017** [GitSquared/edex-ui#1122] [Ideas]
I prefer deb to AppImage because it is easier to distribute and install. AppImage has no desktop shortcut, which is not convenient to use.

**Comment by GitSquared** [comment]
The AppImage should prompt you on launch to install .desktop files for DE integration.

Distro-specific packages are the responsibility of the community, so if you want to package eDEX specifically for your distro, you have to maintain the package yourself. I do this myself for edex & others on Arch's AUR.

**Comment by rainwashed** [comment]
You can just build a desktop entry if you want to have a .desktop file. [AskUbuntu](https://askubuntu.com/questions/281293/creating-a-desktop-file-for-a-new-application)
